SU-SWS/stanford_ssp
# set user name to sunetid, and populate another field with the user di…

# READY FOR REVIEW

# Summary
- set user name to sunetid, and populate another field with the user display name
- Do we need an update hook to create the fields? if the field doesn't exist, nothing happens.

# Needed By (Date)
- october

# Urgency
- low

# Steps to Test
1. check this out
1. delete your user
1. log back in with saml
1. verify you display name is populated in the field.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
